server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -306,8 +306,14 @@
 public class UserVmManagerImpl extends ManagerBase implements UserVmManager, VirtualMachineGuru, UserVmService, Configurable {
     private static final Logger s_logger = Logger.getLogger(UserVmManagerImpl.class);
 
+    /**
+     * The number of seconds to wait before timing out when trying to acquire a global lock.
+     */
     private static final int ACQUIRE_GLOBAL_LOCK_TIMEOUT_FOR_COOPERATION = 3;
-    private static final long GB_TO_BYTES = 1024 * 1024 * 1024;
+    /**
+     * The number of bytes in a GiB.
+     */
+    private static final long GiB_TO_BYTES = 1024 * 1024 * 1024;
```

Comment:
this one is actually self documenting!

Reply:
Sure, removed.

Reply:
i enjoyed the renaming of it very much though ;)

Reply:
come to think about it, and don't take this as criticism on your naming to much, I think constants like this one should be appended with "_FACTOR" to be completely self documenting.

Reply:
no problem. I just renamed to `GiB`, because the previous `GB` prefix that was being used is not accurate. 

Sure the wording "factor" might be more appropriate, but I think we should look for a different solution. We have these constant all over our code base. We should look for a "MetricUtils/UnitUtils" or something like that to convert between different units (bytes to KB, KiB, MB, MiB, GB, GiB, TB, TiB, hertz and Ghz and so on)

Reply:
yes, my remark was about the self documenting quality of the constant name; it is not GiB_TO_BYTES, as it cannot push all information contained in a GiB into a small number of bytes as anyone as ignorant as myself might think. The contant is and can only be used as a factor to convert from GiB to a number of B so it should be called GiB_TO_B_FACTOR or some such name. having said that, another implementator would choose to have a MetricUtil.convertGiBtoB() method.
/me is not a bigot on that difference.